lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -17,7 +17,16 @@ class WP_Webfonts {
 	 * @access private
 	 * @var array
 	 */
-	private static $webfonts = array();
+	private static $registered_webfonts = array();
```

Comment:
Remove `static` here since we're using class members now.

```suggestion
	private $registered_webfonts = array();
```